### PR TITLE
feature(epic-template): Developers should be able to create epics

### LINKED ISSUE #259: EPIC template
**Is your feature request related to a problem? Please describe.**
Currently there is limited direction on how to use and create EPICs within our repos.  

**Describe the solution you'd like**
The node boilerplate and react boilerplate both have an EPIC template. We should bring these over to this repo.

**Describe alternatives you've considered**
We may want to discuss the actual contents of the EPIC template and how to use them in context within each project. 

**Additional context**
Other sample issue templates:
Sample epic template: https://github.com/ethersphere/user-stories/blob/master/.github/ISSUE_TEMPLATE/epic-template.md
Some great examples: https://github.com/devspace/awesome-github-templates

### PR DESCRIPTION
**Pull request by michaelachrisco**
## Changes
1. Add Epic template

## Purpose
The Boilerplate projects have proven that using Epics are a good thing for projects when they are utilized. 

## Approach
Developers should have an example of a working Epic template. 

## Learning
Other sample issue templates:
Sample epic template: https://github.com/ethersphere/user-stories/blob/master/.github/ISSUE_TEMPLATE/epic-template.md
Some great examples: https://github.com/devspace/awesome-github-templates

I would like for a discussion on the content/wording of the template. If anyone has a good idea to add to the template, feel free to chime in! 
Closes #259

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/epic-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what the feature is and what it does.
+
+## Actual/Current Behavior
+A clear and concise description of what is currently happening.
+
+## Steps to Implement
+Add a list of the steps required to implement this feature
+
+## Epic Checklist
+Add a list of issues related to this epic
```
Comment (thomascking) [inline .github/ISSUE_TEMPLATE/epic-issue-template.md:20]: perhaps this should include an example checklist?

**Review by thomascking — APPROVED** [review]
overall lgtm

File: .github/ISSUE_TEMPLATE/epic-issue-template.md (hunk 1 above)
Comment (michaelachrisco) [reply .github/ISSUE_TEMPLATE/epic-issue-template.md:20]: @thomascking Thats a good idea. Something like:

- [ ] Issue #997
- [ ] Issue #998
- [ ] Issue #999

Or maybe just have the issue number:
- [ ] #997
- [ ] #998
- [ ] #999

?

File: .github/ISSUE_TEMPLATE/epic-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+name: Epic Issue Template
```
Comment (fluturecode) [inline .github/ISSUE_TEMPLATE/epic-issue-template.md:2]: Nice work! woo-hoo :tada:

**Review by fluturecode — APPROVED** [review]

**Review by thomascking — APPROVED** [review]
Lgtm

**Pull request merged**